package.json

```diff
@@ -0,0 +1,33 @@
+{
+	"name": "inferno-backend",
+	"version": "1.0.0",
+	"description": "A token distributor for the Inferno Rewards program",
+	"main": "src/index.ts",
+	"repository": "git@github.com:ardriveapp/inferno-backend.git",
+	"author": {
+		"name": "Permanent Data Solutions Inc",
+		"email": "info@ardrive.io",
+		"website": "https://ardrive.io"
+	},
+	"license": "AGPL-3.0",
+	"scripts": {
+		"clean": "rimraf [ lib .nyc_output node_modules coverage ]",
+		"format": "prettier --write \"src/**/*.ts\"",
+		"lint": "eslint . --ext .ts",
+		"lintfix": "eslint . --ext .ts --fix",
+		"typecheck": "tsc --noemit",
+		"build": "yarn clean && tsc --project ./tsconfig.prod.json",
+		"dev": "yarn clean && tsc --project ./tsconfig.prod.json -w"
+	},
+	"devDependencies": {
+		"@yarnpkg/sdks": "^3.0.0-rc.2",
+		"eslint": "^8.13.0",
+		"eslint-config-prettier": "^8.1.0",
+		"eslint-plugin-prettier": "latest",
+		"lint-staged": "^11.0.0",
```

Comment:
lint-staged without husky? also you might want to setup a hook on commits so lint-staged runs like we have on https://github.com/ardriveapp/ardrive-cli/blob/master/.husky/pre-commit